.env.example

```diff
@@ -35,3 +41,10 @@ API_DOMAIN=localhost
 BASE_USER_NAME="Base User"
 BASE_USER_EMAIL=base@example.com
 BASE_USER_PASSWORD=password
+
+AUTH_PASSWORD_LENGTH=30
```

Comment:
Should this be in the committed ENV file?

Reply:
this one is the .env example file